cfgov/jinja2/v1/ask-cfpb/answer-page.html

```diff
@@ -72,14 +72,14 @@ <h1 {% if flag_enabled('HOW_TO_SCHEMA', request) %}itemprop="name"{% endif %}>
                     </span>
                 </button>
             </div>
-        {% if page.related_resource and page.language != 'es' %}
+        {% if page.related_resource and page.related_resource.trans_text(page.language) | striptags %}
```

Comment:
@higginsw I noticed that if a value hasn't been entered for a resource's Spanish text field, the "Next steps" section is output with `None` values on a Spanish page with that resource. Adding an `is not none` condition for the `trans_text` value seems to fix it.

Reply:
Thank you for catching this @virginiacc 
I solved it in the snippet trans method.

Reply:
Works perfectly!